content/4_Gold/Sliding.mdx

```diff
@@ -141,9 +141,40 @@ For each position of the window, we want to compute some information. For exampl
 </Resources>
 
 ### Implementation
+**Time Complexity: ** $\mathcal{O}(N)$
 
-<IncompleteSection />
+<LanguageSection>
+
+<CPPSection>
 
+```cpp
+vector<int> maxSlidingWindow(vector<int>& nums, int k) {
```

Comment:
oops I'm really tired rn but I don't quite understand what this function is doing? Isn't it supposed to return the max consecutive subarray of length k? (why is a `deque` needed?)

Reply:
you must be really tired :P

Reply:
okokokokok in my defense that was 7:30am LMAOO